WebRoot/skins/_base/base3/skin.html

```diff
@@ -34,6 +34,7 @@
 					<tr role='banner'>
 						<td id='skin_spacing_logo'>
 							<div id='skin_container_logo' class='skin_container'></div>
+							<div id="headerTitle"></div>						

```

Comment:
Can we set 'ScreenReaderOnly' class, 'role' and 'aria-level' attributes here instead of setting by javascript  at https://github.com/Zimbra/zm-web-client/pull/768/files#diff-8b6cdbb5f510468ca8dff8847170767516e52e66252e38a4446fb4b81e3c38bdR1854

Reply:
sure we can do that as well.

Reply:
done